zingocli/src/lib.rs

```diff
@@ -513,6 +514,11 @@ fn dispatch_command_or_start_interactive(cli_config: &ConfigTemplate) {
 
 /// TODO: Add Doc Comment Here!
 pub fn run_cli() {
+    // install default crypto provider (ring)
+    rustls::crypto::ring::default_provider()
+        .install_default()
+        .expect("Failed to install rustls crypto provider");
```

Comment:
In general `expect` and `unwrap` are not allowed in our codebase.  Should this function return Result?